divi/src/UtxoCheckingAndUpdating.cpp

```diff
@@ -10,28 +10,35 @@
 #include <undo.h>
 #include <chainparams.h>
 #include <defaultValues.h>
+#include <IndexDatabaseUpdates.h>
 
 extern BlockMap mapBlockIndex;
 
-void UpdateCoinsWithTransaction(const CTransaction& tx, CCoinsViewCache& inputs, CTxUndo& txundo, int nHeight)
+uint256 BlockUtxoHasher::GetUtxoHash(const CTransaction& tx) const
+{
+    return tx.GetHash();
+}
```

Comment:
Why inherit again from the TransactionUtxoHasher when the class itself already provides the necessary functionality? Seems like that would add indirection